README.md

```diff
@@ -43,43 +45,43 @@ Assets in this category may be refined into more specific Layer 2 controls, or c
 
-Activities in the Policy layer provide risk-informed governance rules tailored to an organization. They are typically informed by risks or based on best practices and industry standards.
+Activities in the Policy layer provide risk-informed governance rules that while based on best practices and industry standards are tailored to an organization.
 
-Layer 3 controls are typically developed by an organization for its own purposes, to compile into organizational policies. Policies cannot be properly developed without consideration for organization-specific risk appetite and risk-acceptance.
+Layer 3 controls are typically developed by an organization to compile into organizational policies. Policies cannot be properly developed without consideration for organization-specific risk appetite and risk-acceptance.
 
 These policy documents may be referenced by other policy documents, or used as a starting point for Layer 4 assessments.
 
 ### Layer 4: Evaluation
 
 Activities in the Evaluation layer provide inspection of code, configurations, and deployments. Those elements are part of the _software development lifecycle_ which is not represented in this model.
 
-Evaluation activities may be built based on outputs from layers 2 or 3. While automated assessments are often developed by vendors or industry groups, proper evaluation should also be able to ingest organizational policies to custom-tailor the assessment to the needs of the compliance program.
+Evaluation activities may be built based on outputs from layers 2 or 3. While automated assessments are often developed by vendors or industry groups, robust evaluation should be informed by organizational policies in order to custom-tailor the assessment to the needs of the compliance program.
 
 #### Layer 4 Schema
 
-The SCI [Layer 4 Schema](./schemas/layer-4.cue) provides methods for expressing Layer 4 evaluation results in a machine readable format.
+The SCI [Layer 4 Schema](./schemas/layer-4.cue) describes the machine-readable format of Layer 4 evaluation results.
 
 The schema allows evaluations to be mapped to Layer 2 controls by their unique identifiers.
 
-The SCI go module provides Layer 2 support for writing assessments that can write results in this schema.
+The SCI go module provides Layer 2 support for writing assessments that can write results in this schema. The work to add go module Layer 4 support for reading and writing assessments data is planned in [issue 30](https://github.com/revanite-io/sci/issues/30), but not completed.
 
 ### Layer 5: Enforcement
 
-Activities in the Enforcement layer provide prevention or remediation based on assessment findings.
-
-Enforcement actions should be dictated by Layer 3 policies, and act on information gathered by Layer 4 evaluations.
+Activities in the Enforcement layer provide prevention or remediation based on assessment findings gathered by Layer 4 evaluations. Enforcement actions should also be further guided by Layer 3 policies.
```

Comment:
```suggestion
Activities in the Enforcement layer provide prevention or remediation. These enforcement actions should be guided by Layer 3 policies and based on assessment findings from Layer 4 evaluations.
```

Reply:
So much better, thank you for the rewrite!